web/themes/new_weather_theme/templates/wfo-info/node--wfo-info.html.twig

```diff
@@ -1,41 +1,111 @@
-<h1>
-    WFO Information Page
-</h1>
-
-<h2>Image</h2>
-{{content.field_wfo_image}}
-
-<h2>WFO</h2>
-<dl>
-    {% for key,val in wfo %}
-        <dt>{{key}}</dt>
-        <dd>{{val}}</dd>
-    {% endfor %}
-</dl>
-
-<h2>About</h2>
-{{content.body}}
-
-<h2>Phone number</h2>
-<dl>
-    {% for key,val in phone %}
-        <dt>{{key}}</dt>
-        <dd>{{val}}</dd>
-    {% endfor %}
-</dl>
-
-<h2>Address</h2>
-<dl>
-    {% for key,val in address %}
-        <dt>{{key}}</dt>
-        <dd>{{val}}</dd>
-    {% endfor %}
-</dl>
-
-<h2>External links</h2>
-<dl>
-    {% for key,val in external_links %}
-        <dt>{{key}}</dt>
-        <dd>{{val}}</dd>
-    {% endfor %}
-</dl>
+<div class="wx-wfo-info">
+  {% if content.field_wfo_image is not empty %}
+    <div class="wx-wfo-image grid-container padding-0 tablet:padding-x-2 desktop:padding-x-4">
+      {{ content.field_wfo_image }}
+    </div>
+  {% endif %}
+
+  <div class="grid-container">
+    <h1 class="text-normal text-primary-darker">
+      {{ "@wfoName (@wfoCode) National Weather Service Office" | t({ "@wfoName": wfo.name, "@wfoCode": wfo.code }) }}
+    </h1>
+
+    <div class="grid-row">
+      {% set hasAbout = not (content.body.isEmpty or content.body.0 is not defined) %}
+      {% if hasAbout %}
+      <div class="grid-col-12 tablet:grid-col-4">
+        <h2 class="text-normal text-primary-darker">{{ "About" | t }}</h2>
+        {{content.body}}
+      </div>
+      {% endif %}
+
+      <div class="grid-col-12{% if hasAbout %} tablet:grid-col-8 tablet:padding-left-2{% endif %}">
+        <figure class="margin-0">
+          <img src="/{{ directory }}/assets/images/wfos/{{ wfo.code }}.png" alt="">
+          <figcaption>{{ "Coverage area for @wfo WFO" | t({ "@wfo": wfo.name })}}</figcaption>
+        </figure>
+      </div>
+    </div>
+
+    {#
+      TODO: This is all placeholder stuff. When it comes out in favor of real
+      stuff, make sure we add things to the translations list appropriately.
+    #}
+    <div class="grid-row">
+      <div class="grid-col-12 tablet:grid-col-6">
```

Comment:
On mobile the spacing between the sections is a uneven. Could we make it so there is 4 units of space between the between the bottom of one section and the start (border) of the next

![Screenshot of uneven spacing](https://github.com/user-attachments/assets/c82b1f5b-4a40-4a40-aff2-cd3aa0bf8207)

Reply:
👍 I added `margin-top-4 padding-top-4` to all the headings and that seemed to accomplish it?

Reply:
Yep, that looks great, only one that sticks out is the list under Local Expertise, so I think we add `margin-bottom-0` to that `ul` we should be good to go